packages/trader/src/App/Components/Elements/PositionsDrawer/positions-drawer-card.jsx

```diff
@@ -45,6 +45,7 @@ const PositionsDrawerCard = ({
             />
         </div>
     );
+    const fallback_result = (profit_loss < 0) ? 'lost' : 'won';
```

Comment:
do we need `+profit_loss` or we know it's a number?

Reply:
It should already be sent as `number` from `portfolio-store`.
Refer to [L99](https://github.com/binary-com/deriv-app/blob/dev/packages/trader/src/Stores/Modules/Portfolio/portfolio-store.js#L99).